packages/components/src/Loading/LoadingSpinner/_docs/LoadingSpinner.mdx

```diff
@@ -29,14 +29,17 @@ When inside a button, it is intended to have the same color as the label text.
 
 ## API
 
+
+### Size
+
+Generally use `"md"` (default value) unless spinner is inside a form field, in which case use `sm`. The `xs` size is for buttons or content dense layouts.
+
+<Canvas of={LoadingSpinnerStories.Size} />
+
 ### ClassNameOverride
 
 Override styles such as the colour of the spinner using `classNameOverride`.
 
 <Canvas of={LoadingSpinnerStories.ClassNameOverride} />
 
-### Size
-
-Generally use `"md"` (default value) unless spinner is inside a form field.
-
-<Canvas of={LoadingSpinnerStories.Size} />
+You can also use this to target the `--loading-spinner-size` CSS variable if required. Note that this will not affect the stroke width of the svg so use with caution.
```

Comment:
I'm a little dubious in terms of how much we want to promote this so curious for thoughts @dougmacknz @caseysinclair. Happy to leave it out of here and just have it as a thing folks can do if they need an escape hatch :)

Reply:
Yeah I'd leave this out

Reply:
Agreed, we should leave this out because we aren't encouraging the use of manipulating the size (unlike new Icon).

Reply:
Cool 👍